user-manager/src/main/java/com/qpay/usermanager/mapper/AuthenticationUserMapper.java

```diff
@@ -0,0 +1,59 @@
+package com.qpay.usermanager.mapper;
+
+import com.qpay.libs.models.UserType;
+import com.qpay.usermanager.model.dto.authuser.AuthenticationUserCreation;
+import com.qpay.usermanager.model.dto.authuser.AuthenticationUserModification;
+import com.qpay.usermanager.model.dto.customer.CustomerModification;
+import com.qpay.usermanager.model.dto.merchant.MerchantCreation;
+import org.springframework.stereotype.Component;
+
+@Component
+public class AuthenticationUserMapper {
```

Comment:
Too many methods. Merge them. You don't need separate methods with and without password. You have "if"